L1Trigger/Phase2L1ParticleFlow/python/L1MetPfProducer_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-L1MetPfProducer = cms.EDProducer("L1MetPfProducer",
-                                 L1PFObjects = cms.InputTag("l1ctLayer1","Puppi"),
+l1tMETPFProducer = cms.EDProducer("L1MetPfProducer",
```

Comment:
Duplicate of
cfipython/L1Trigger/Phase2L1ParticleFlow/L1METPFProducer_cfi.py

with the same search path

Reply:
Done